test/test_datasets.py

```diff
@@ -2463,5 +2463,61 @@ def _meta_to_split_and_classification_ann(self, meta, idx):
         return (image_id, class_id, species, breed_id)
 
 
+class StanfordCarsTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.StanfordCars
+    REQUIRED_PACKAGES = ("scipy",)
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(train=(True, False))
+
+    def _inject_fake_data(self, tmpdir, config):
+        import scipy.io as io
+        from numpy.core.records import fromarrays  # to record arrays similar to matlab format
+
+        train = config["train"]
+        num_examples = 5
+        root_folder = os.path.join(tmpdir)
```

Comment:
This is not doing anything. If you want to have the name "mapping" use

```suggestion
        root_folder = tmpdir
```

Reply:
oh yes,